lib/constants.js

```diff
@@ -13,4 +13,22 @@ c.SUPPORTED_NETWORK_IDS = [
 
 c.DEFAULT_NETWORK_ID = c.MAINNET_NETWORK_ID
 
+c.ERROR_LOG_LEVEL = 'error'
+c.WARN_LOG_LEVEL = 'warn'
+c.INFO_LOG_LEVEL = 'info'
+c.DEBUG_LOG_LEVEL = 'debug'
+
+c.SUPPORTED_LOG_LEVELS = [
+  c.ERROR_LOG_LEVEL,
+  c.WARN_LOG_LEVEL,
+  c.INFO_LOG_LEVEL,
+  c.DEBUG_LOG_LEVEL
+]
+
+c.DEFAULT_LOG_LEVEL = c.INFO_LOG_LEVEL
+
+c.DEFAULT_RPC_PORT = 8545
+
+c.DEFAULT_RPC_ADDR = 'localhost'
```

Comment:
Just as a note: we should structure this also a bit into groups and add a comment header to a group, not super high priority though, just when anyone thinks on it while maybe adding some new constants.